tests/fixtures/event/proposal.py

```diff
@@ -0,0 +1,190 @@
+"""
+Proposal for an
+edx.problem.hint.feedback_displayed event fixture definition
+"""
```

Comment:
This is an example for the proposition of event fixture definition.
One major drawback is that it's quite similar to the current solution - just without inheritance, so I'm not sure if it's more maintainable.